sqladmin/templates/list.html

```diff
@@ -164,6 +152,18 @@ <h3 class="card-title">{{ model_view.name_plural }}</h3>
           </a>
         </li>
       </ul>
+      <div class="dropdown text-muted">
+        <a href="#" class="btn btn-sm btn-light dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+          {{ request.query_params.get("pageSize") or model_view.page_size }} / Page
+        </a>
+        <div class="dropdown-menu">
+          {% for page_size_option in model_view.page_size_options %}
+          <a class="dropdown-item" href="{{ request.url.include_query_params(pageSize=page_size_option) }}">
+            {{ page_size_option }} / Page
```

Comment:
My only comment is about the wording here. Can we do something like `Show <x> items`?
I think that would be more readable.

Reply:
Yes, I'll add it.